authentication/using-service-accounts-in-applications.adoc

```diff
@@ -10,8 +10,15 @@ include::modules/service-accounts-overview.adoc[leveloffset=+1]
 
 include::modules/service-accounts-default.adoc[leveloffset=+1]
 
+// remove these links for 4.12+
+
+.Additional resources
+
+* For information on requesting bound service account tokens, see xref:../authentication/bound-service-account-tokens.html#bound-sa-tokens-configuring_bound-service-account-tokens[Using bound service account tokens]
+
+* For information on creating a service account token secret, see see xref:../nodes/pods/nodes-pods-secrets.html#nodes-pods-secrets-creating-sa_nodes-pods-secrets[Creating a service account token secret].
+
 include::modules/service-accounts-creating.adoc[leveloffset=+1]
 
```

Comment:
This modules/service-accounts-creating.adoc still contains the auto-generated legacy token secret `robot-token-f4khf` (FYI `robot-token-f4khf` is expected, it is not an auto-generated legacy token secret):
```
Mountable secrets: 	robot-token-f4khf
                   	robot-dockercfg-qzbhb

Tokens:            	robot-token-f4khf
                   	robot-token-z8h44
```
In 4.11 and above, it should look like:
```
Mountable secrets:   robot-dockercfg-sfjnj
Tokens:              robot-token-w2zjc
```